tests/test_analysis.py

```diff
@@ -90,7 +90,7 @@ class fano_factor_tests(unittest.TestCase):
     def test_get_fano_factor_values_per_condition(self): 
```

Comment:
change the function name to reflect the new test?

Reply:
```suggestion
    def test_get_unit_spiking_mean_variance(self): 
```